mix.exs

```diff
@@ -52,22 +52,37 @@ defmodule Barrel.Mixfile do
 
   defp deps do
     [
-      {:barrel_ex_logger, git: "https://github.com/barrel-db/barrel_ex_logger.git"},
+      {:lager_logger, "~> 1.0"},
       {:exrm, "~> 0.18.1"},
       {:barrel_nifs, path: "support/barrel_release_plugin"},
-      {:hooks,  "~> 1.1.1"},
-      {:mochicow, "~> 0.6.4"},
-      {:snappy, "~> 1.1"},
-      {:ucol, "~> 2.0"},
+      {:metrics, "~> 1.0.1", manager: :rebar, override: true},

```

Comment:
rebar3 is the default and doesn't work. meanwhile @sivsushruth is working
on a minimal case
On Sun, 17 Jul 2016 at 15:25, José Valim <notifications@github.com> wrote:

> In mix.exs
> <https://github.com/barrel-db/barrel-platform/pull/52#discussion_r71082324>
> :
>
> >        {:exrm, "~> 0.18.1"},
> >        {:barrel_nifs, path: "support/barrel_release_plugin"},
> > -      {:hooks,  "~> 1.1.1"},
> > -      {:mochicow, "~> 0.6.4"},
> > -      {:snappy, "~> 1.1"},
> > -      {:ucol, "~> 2.0"},
> > +      {:metrics, "~> 1.0.1", manager: :rebar, override: true},
>
> Shouldn't those be manager: :rebar3 so we use latest rebar?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/barrel-db/barrel-platform/pull/52/files/ae1abcb6b9f341e4da37e2448e75098553c1ea6f#r71082324>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAA4on1t9fnTh811uEwHK1_rP5_GYoQfks5qWi1MgaJpZM4JE9kW>
> .
>